app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
I was always under the impression that this allowed method calls on the Hostgroup object, yet there is no subscription_manager_configuration_url method on this object.  I'm fairly certain this change is fine (as its tthere on host) but can you explain exactly how this works?